app/src/main/kotlin/com/hippo/ehviewer/ktor/CronetEngine.kt

```diff
@@ -65,11 +62,11 @@ object CronetEngine : HttpClientEngineBase("Cronet") {
                     info.toHttpResponseData(
                         requestTime = requestTime,
                         callContext = callContext,
-                        responseBody = GlobalScope.writer(callContext) {
-                            pool.useInstance {
-                                chunkChan = Channel()
-                                request.read(it)
-                                chunkChan.consumeEach { buffer ->
+                        // Don't use the same coroutine context as the response, as it will block response processing.
+                        responseBody = writer {
```

Comment:
This `WriterJob` will never be cancelled if request is cancelled when processing processing

Reply:
> ~This `WriterJob` will never be cancelled if request is cancelled when processing response~ It doesn't matter as both `CronetRequest` and `KtorRequest` will be released so no res-leak

`DirectByteBufferPool` instance will leak